src/app/basics/getting-started/record-your-playwright-tests/page.md

```diff
@@ -22,33 +22,39 @@ It will create an API key and guide you through each step.
 {% tab %}
 
 ```sh
-npm install @replayio/playwright -D
+npm install --save-dev @replayio/playwright
 ```
 
 {% /tab %}
 {% tab %}
 
 ```sh
-yarn add @replayio/playwright -D
+yarn add --save-dev @replayio/playwright
 ```
 
 {% /tab %}
 {% tab %}
 
 ```sh
-pnpm install @replayio/playwright -D
+pnpm install --save-dev @replayio/playwright
 ```
 
 {% /tab %}
 {% tab %}
 
 ```sh
-bun install @replayio/playwright -D
+bun install --save-dev @replayio/playwright
```

Comment:
what's wrong with `bun install -D @replayio/playwright` and others as well?

Reply:
Docs had this syntax:
npm install @replayio/playwright -D

Our orange new team flow had this syntax:
npm install --save-dev @replayio/playwright

So this change is aligning with the more recent and baked-in style we see in the app. I also think there's benefit to writing flags out to make them easier to read and understand, but that's a subjective copy thing.

Reply:
That's fine.

Reply:
I think Bun should be
```suggestion
bun add --dev @replayio/playwright
```